modules/authorization/appRoleAssignments.bicep

```diff
@@ -14,12 +14,15 @@ var _name = guid(resourceGroup().id, userName, appRoleDefinitionId)
 
 resource managedApp 'Microsoft.Solutions/applications@2021-07-01' existing = {
   name: appName
-  scope: resourceGroup(sub, appResourceGroupName)
+  // scope: resourceGroup(sub, appResourceGroupName)
+  // this scope is implicit since the module itself is already scope to this RG
 }
 
 resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-10-01-preview' = {
   name: _name
-  scope: managedApp
+  scope: managedApp // this "scope" property only works if the resource is within the same scope as the module
+  // even though you are passing the same sub and RG info, bicep sees it as a different RG. Commenting out 
+  // line 17 resolves this issue
```

Comment:
If the scope on the module is removed, line 23 of other file, this would then need to be defined right?